site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -60,7 +61,7 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
   // - users are loading or
   // - the user can edit the users but the roles are loading
   const isLoading =
-    usersState.matches("gettingUsers") ||
+    usersState.matches("users.gettingUsers") ||
```

Comment:
Was this just not working before? How come we have to add `users` now?

Reply:
I put all the existing states inside a "users" state node (submachine) so that the count-related states can run in parallel to them